New README

mmattel opened the pull request:
As discussed on talk, this is a new README.md reflecting the recent changes regarding the processes and management around ``ui-bundle.zip``.

settermjd commented on README.md:
```diff
@@ -1,3 +1,117 @@
+When compiling the documentation in the ``docs repro``, this ``ui-bundle.zip`` file is, if not manually 
+defined differently, automatically used.
+
+If you want to see the result of your unmerged UI changes made in a compiled documentation, 
```
You can run `gulp preview` instead, which generates and serves a preview site, based on the existing configuration.

settermjd commented on README.md:
```diff
@@ -1,3 +1,117 @@
+option in your ``antora``  command to use it instead the published version. 
+See the [README][link-README] in the ``docs reposirory`` for further information how to compile the documentation.
+
+## Package Creation for Use with Antora
```
This isn't necessary, as this is ownCloud-specific and the bundle's automatically built as part of the CI/CD build pipeline.

settermjd commented on README.md:
```diff
@@ -1,3 +1,117 @@
+
+## Prerequisites
+
+To create a local version of a ``ui-bundle.zip``, you need to install the following software on your computer.
```
As it's not necessary to generate a ui-bundle archive, I suggest changing the wording to refer to users being able to run `gulp preview` instead.

mmattel commented:
- There is a fix that ``gulp preview`` is running at all - seems that nobody has tried it before..., see https://github.com/owncloud/docs-ui/pull/13

- ``gulp preview`` is a possibility but not a solution, 
as a developer, I want to see the result on the documentation as it is rendered, built by the antora command

- Only a build with the ``ui-bundle.zip`` via the ``antora`` build command will show the final result

See the difference

**gulp pack**
![image](https://user-images.githubusercontent.com/3321281/48717556-f72c1900-ec19-11e8-9a16-16a13705a8d2.png)

**antora build**
![image](https://user-images.githubusercontent.com/3321281/48717577-03b07180-ec1a-11e8-98c7-04cac63b6692.png)

tboerger commented:
> * There is a fix that `gulp preview` is running at all - seems that nobody has tried it before..., see #13

That worked fine for me...

> * `gulp preview` is a possibility but not a solution,
>   as a developer, I want to see the result on the documentation as it is rendered, built by the antora command

It is a solution, the developer who updates the theme doesn't need the full blown documentation to check UI adjustments.

mmattel commented:
This is an improvement based on a discussion with @settermjd.
Referencing a README change in https://github.com/owncloud/docs/pull/403 to be inline.

I updated the readme for `gulp preview` and added a note to look at the docs readme if you want to check changes with full content.

mmattel commented on README.md:
```diff
@@ -12,47 +12,45 @@
 
-It contains the content that is valid for the whole documentation like `header`, `footer` ect. and is embedded during the documentation build process automatically.
+If you want to make changes, create a "_feature_" branch off of master, make the required changes, and then create a Pull Request (PR) against the _master_ branch.
+If the PR is accepted and merged, a new `ui-bundle.zip` package file will be created and published to https://minio.owncloud.com/documentation/ui-bundle.zip.
```
I would say: `created` --> `created automatically`
because else it is not clear who is doing that

settermjd replied:
I don't totally agree, but I see your point.

settermjd replied:
Just corrected it. Thanks.

mmattel commented on README.md:
````diff
@@ -12,47 +12,45 @@
 
 ```
- $ gulp pack
+ $ gulp preview
````
question, does gulp preview show the content without it has been packed first ?

settermjd replied:
`gulp preview` generates a demo version of the site.

settermjd replied:
It doesn't generate a ui bundle archive.

tboerger commented:
And if you drop the `$ ` from the commands it's even easier to copy and past them ;)

mmattel commented:
will do

tboerger commented:
I personally prefer the links inline instead of the separate list of links, but that's not my decision I guess.

mmattel commented:
hmmm, the links are all inline, it is just the definition of the links on top used in the text flow when referenced. the definition list is not rendered.

tboerger commented:
I know that it won't be rendered, but I personally prefer it inline, without the separate list of links.
